tests/Console/Commands/FindMissingTranslationStringsTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace CodingSocks\LostInTranslation\Tests\Console\Commands;
+
+use CodingSocks\LostInTranslation\LostInTranslationServiceProvider;
+use Illuminate\Config\Repository;
+use Illuminate\Support\Facades\Artisan;
+use Orchestra\Testbench\TestCase;
+
+class FindMissingTranslationStringsTest extends TestCase
```

Comment:
You could extend `CodingSocks\LostInTranslation\Tests\TestCase` and have the `getPackageProviders` already set up.

Reply:
Whoops, good catch.